### ISSUE TITLE
Standardize All grep Commands to GNU Grep 3.0+

### ISSUE THREAD

**Issue by hash-man-sabbath** [jonathan-hurley/poker-mavens#1]
change all ggrep to grep, remove -E from the commands line.

this is what I had to do in order for the scripts to work, and now its only showing player hands, nothing else

**Comment by jonathan-hurley** [comment]
Yeah, I think `ggrep` and `egrep` were from the initial days of trying to build out these commands on a mac. Today, `grep` should be GNU grep, but also still requires the `-E` for it to enable expanded regex. 

On macos, I have this:
```
# grep --version
grep (GNU grep) 3.11
Packaged by Homebrew
```

And on Windows (via Git bash), I have this:
```
$ grep --version
grep (GNU grep) 3.0
Copyright (C) 2017 Free Software Foundation, Inc.
```

I'm happy to remove instances of `ggrep` and `egrep`, but `-E` is still needed.

**Comment by jonathan-hurley** [comment]
I pushed https://github.com/jonathan-hurley/poker-mavens/commit/24661198d470213f079bbb28a5da9c2d4ce5ecc1 which includes:
- standardization on GNU `grep`
- fixes to scripts
- additional SQL storage / state generation
- UI / JQuery / Bootstrap updates & fixes

The database changed, so you should delete the DB and try to regenerate. I ran through my sample data and everything came out clean; all 4 `web` files were generated.
